test/WebJobs.Extensions.Tests/Extensions/ApiHub/ApiHubBindingEndToEndTests.cs

```diff
@@ -243,6 +323,16 @@ private void DeleteTestFiles(string path)
             }
         }
 
+        private async Task DeleteApiHubBlobsAsync()
+        {
+            await GetBlobReference("ImportTestJob").DeleteIfExistsAsync();
```

Comment:
You should just do a blob directory listing on apihubs/{siteName} and iterate over all blobs found and delete them. That way you don't have to remember to add string literals for each new function (they can get out of sync).